src/app/app.component.html

```diff
@@ -12,15 +12,16 @@
     <wave-layer-list class="mat-elevation-z4" [height]="middleContainerHeight$ | async"></wave-layer-list>
 
     <div class="mid-container" [style.height.px]="middleContainerHeight$ | async">
-        <wave-ol-map #map>
```

Comment:
If we add another map we have to indicate the type in the container name.

Reply:
Maybe you can clarify your suggestion, because I think I do not get your point.
My thoughts by now:
 1) we only use one map component
 2) #map is used to access the component from `app.component`.
 3) I don't know the _type_ of a map. And usually I resist coding types into variable names.

Reply:
oh i was thinking of cesium, harp.gl, ... as alternative to openlayers. Then we would have wave-ol-map, wave-cesium-map, ...

Reply:
I think this would be a major change that needs a lot of refactoring then. Now, it's the only option and it is more than just one ol map - it is potentially a grid of maps. So I would leave it like this for now.